src/js/components/Image/__tests__/Image-test.js

```diff
@@ -28,3 +29,15 @@ test('Image fit renders', () => {
   const tree = component.toJSON();
   expect(tree).toMatchSnapshot();
 });
+
+opacityTypes.forEach(opacity => {
+  test('Image opacity renders', () => {
```

Comment:
can we add the type of opacity in the test string? it will be easier to debug when something will break